### DISCUSSION TITLE
Is the default config not calling `cabalfmt` correctly ?

### DISCUSSION THREAD

**Discussion by leana8959** [mrcjkb/haskell-tools.nvim#291] [Q&A] [answered]
I noticed that, in the settings, if I don't override `cabalFormattingProvider = 'cabalfmt'` with `cabalFormattingProvider = 'cabal-fmt'` (with the dash), I can't format cabal files.

Is this intended ? Am I installing `cabal-fmt` wrong ?
I can definitely create an alias, but this feels like a typo in the default settings to me.

Thanks :)

**Comment by mrcjkb** [comment]
Hey :wave: 

I think you might be right in your assumption that it's a typo (or the setting has been renamed).
I'll look into it.

**Reply by leana8959** [reply]
Cool, thank you ^^

**Reply by mrcjkb** [reply, marked as answer]
I've removed the default setting, so it should fall back to haskell-language-server's default.

**Reply by leana8959** [reply]
I see, thank you for the help !!
